phoebe/backend/universe.py

```diff
@@ -301,7 +307,7 @@ def update_positions(self, time, xs, ys, zs, vxs, vys, vzs,
                                  ds=ds, Fs=Fs, ignore_effects=ignore_effects)
 
 
-    def populate_observables(self, time, kinds, datasets, ignore_effects=False):
+    def populate_observables(self, time, kinds, datasets, ignore_effects=False, **kwargs):
```

Comment:
is this (`**kwargs` here and in calls to `populate_observable`) used in this PR?

Reply:
Corrected. It was in a version with observe(); obsolete.